packages/classes/src/lib/utils/instantiate.util.ts

```diff
@@ -69,7 +69,9 @@ export function instantiate<TModel extends Dictionary<TModel>>(
     // if is Date, assign a new Date value if valueAtKey is defined, otherwise, undefined
     if (isDateConstructor(metaResult)) {
       const value = isDefined(valueAtKey)
-        ? new Date(valueAtKey as number)
```

Comment:
issue: we need to copy this same logic to `pojos/instantiate.util.ts` as well

Reply:
Added to pojos.